server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTableProcessor.java

```diff
@@ -172,7 +173,7 @@ private RrtExecutionResult getRrtExecutionResult(Mail mail, MailAddress recipien
             .collect(Guavate.toImmutableList());
         
         if (!addressWithoutDomains.isEmpty()) {
-            final String defaultDomain = getDefaultDomain(domainList);
+            final Domain defaultDomain = getDefaultDomain(domainList);
```

Comment:
final variable spotted

Reply:
I refrain from changing things that are not the intent of a changeset. I don't see any valid reason the remove final from there in this changeset.